src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -71,15 +112,75 @@ public void close() throws Exception {
     // Mostly for JUnit tests
     motor.close();
     motor2.close();
-    
   }
 
-  @Override
-  public void periodic() {
-    intakeMode.setString((isConeMode) ? "Cone Mode" : "Cube Mode");
-    // This method will be called once per scheduler run
+  public void changeFlightState() {
+    // Check if sensors are online
+    boolean coneSensorUp = timeOfFlightSensor.isSensor0Connected();
+    boolean cubeSensorUp = timeOfFlightSensor.isSensor1Connected();
```

Comment:
Now that getDistanceX() will return -1 if the sensor isn't up, we should just save the getDistanceX() values, and then derive the coneSensorUp by checking if we got -1. That way we avoid the possibility of using different measurements from the ToF thread.
```suggestion
    boolean coneSensorUp = coneDistance != -1;
    boolean cubeSensorUp = cubeDistance != -1;
```